up.go

```diff
@@ -75,13 +75,32 @@ var (
 	unsafeMode   = pflag.Bool("unsafe-full-throttle", false, "enable mode in which command is executed immediately after any change")
 	outputScript = pflag.StringP("output-script", "o", "", "save the command to specified `file` if Ctrl-X is pressed (default: up<N>.sh)")
 	debugMode    = pflag.Bool("debug", false, "debug mode")
+	versionFlag = pflag.Bool("version", false, "display version and credits")
 )
 
 func main() {
 	// Handle command-line flags
 	pflag.Parse()
 
 	log.SetOutput(ioutil.Discard)
+	if *versionFlag {
+		os.Stdout.WriteString("up: Ultimate Plumber v" + version + " https://github.com/akavel/up\n")
+		os.Stdout.WriteString("up: \n")
+		os.Stdout.WriteString("up: Dependencies:\n")
+		os.Stdout.WriteString("up: \tTcell\n")
+		os.Stdout.WriteString("up: \t\thttps://github.com/gdamore/tcell\n")
+		os.Stdout.WriteString("up: \t\tLicense: Apache-2.0, ECL-2.0\n")
```

Comment:
As I wrote already in #34, please provide links to the license files in github repositories of the projects. For example, I don't know what "ECL-2.0" means. I would like to have some link, like:

    ECL-2.0  https://github.com/gdamore/tcell/.../LICENSE#L12345

which I could easily click and see the full text of the license.